app/views/users/show.html.erb

```diff
@@ -0,0 +1,84 @@
+<div class="container mt-4">
+  <h2 class="text-info">Profile Page</h2>
+  <div>
+    <p class="lead">User: <%= @user.name %></p>
+  </div>
+</div>
+
+<div class="container">
+  <div class="row">
+    <% @user.attended_events do |e| %>
+      <%= e.description %>
+    <% end %>
+  </div>
+</div>
+
+<div class="container">
+  <h3 class="text-info">Upcoming events</h3>
+
+  <table class="table table-striped border">
+    <tr>
+        <th class="table-secondary">
+            Name
+        </th>
+        <th class="table-secondary">Description</th>
+        <th class="table-secondary">Date</th>
+        <th class="table-secondary">Creator</th>
+        <th class="table-secondary"></th>
+    </tr>
+    <% @upcoming_events.each do |e| %>
```

Comment:
- [ ] There should be no logic at all( if else, each, etc) in the views.
View logic should be inside the pertinent helper.